CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 - remove the `package:args` dependency #94
 - replace the `package:usage` by `package:uuid` #94
 - move the `package:pedantic` to dev depencies #94
+- Added GH Action Changelog verifier #95
+
+# `package:sentry` changelog
```

Comment:
This seem out of place now. Since all changes are related to `package:sentry` but are above this section.